src/main/java/picard/illumina/IlluminaBasecallsToSam.java

```diff
@@ -312,21 +297,14 @@ private void initialize() {
         }
 
         if (USE_NEW_CONVERTER) {
-            final MetricsFile<BarcodeMetric, Integer> metrics = getMetricsFile();
-
-            if (METRICS_FILE != null) {
-                IOUtil.assertFileIsWritable(METRICS_FILE);
-            } else {
-                log.warn("No METRIC_FILE specified. Barcode metrics will not be generated.");
-            }
+            if (BARCODES_DIR == null) BARCODES_DIR = BASECALLS_DIR;
             basecallsConverter = new NewIlluminaBasecallsConverter<>(BASECALLS_DIR, BARCODES_DIR, LANE, readStructure,
                     barcodeSamWriterMap, true, Math.max(1, MAX_READS_IN_RAM_PER_TILE / numOutputRecords),
                     TMP_DIR, NUM_PROCESSORS,
                     FIRST_TILE, TILE_LIMIT, new QueryNameComparator(),
                     new Codec(numOutputRecords),
                     SAMRecordsForCluster.class, bclQualityEvaluationStrategy,
-                    this.APPLY_EAMSS_FILTER, INCLUDE_NON_PF_READS, IGNORE_UNEXPECTED_BARCODES, MAX_NO_CALLS,
-                    MAX_MISMATCHES, MIN_MISMATCH_DELTA, MINIMUM_QUALITY, metrics, METRICS_FILE);
+                    this.APPLY_EAMSS_FILTER, INCLUDE_NON_PF_READS, IGNORE_UNEXPECTED_BARCODES);
```

Comment:
No your doing, but we have `this. APPLY_EAMSS_FILTER` and `INCLUDE_NON_PF_READS`, can we remove `this.`?